jme3-core/src/main/java/com/jme3/effect/ParticleEmitter.java

```diff
@@ -284,6 +284,20 @@ public ParticleEmitter(String name, Type type, int numParticles) {
         control = new ParticleEmitterControl(this);
         controls.add(control);
 
+        this.initParticleMesh();
+        this.setNumParticles(numParticles);
+//        particleMesh.initParticleData(this, particles.length);
+    }
+
+    /**
+     * For serialization only. Do not use.
+     */
+    protected ParticleEmitter() {
+        super();
+        setBatchHint(BatchHint.Never);
+    }
```

Comment:
Why did you move the protected constructor?

Reply:
I inserted the method after the default constructor. I don't know why github points out the differences so weird for this point. If you see the whole file you can check that everything is ok.

Reply:
GitHub's diff algorithm is unhelpful sometimes, and I guess this is one of those times. Sorry for my confusion!